modules/rtm/src/RTM/pro2s/pro42s.f90

```diff
@@ -0,0 +1,22 @@
+subroutine pro42s( &
```

Comment:
I have unit tests that check that all of these functions give reasonable output (e.g. no values greater than 1 or less than 0; matrices/vectors of the right dimensions). The output plots also look reasonable.